src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -33,73 +39,104 @@ export const ArtworkImageBrowser = (props: ArtworkBrowserProps) => {
   )
 }
 
+/*
+ * Returns null if no window for SSR
+ */
+const isClient = typeof window !== "undefined"
+const Flickity = isClient ? require("react-flickity-component") : () => null
```

Comment:
Rather than duplicating this logic, it might make more sense to move it inside of a factory function inside of `Carousel`, something like: 

```js
const { Flickity } = getFlickity() 
```
and inside of there it handles this bit of duplicated logic. 

The ideal is being able to just use `Carousel` (or Flickity if its separate from the Carousel component) with as little overhead as possible.

Reply:
A good way to think about this is a new file -- `Components/Flickity` -- serves as a factory function (or something) handling setup and so on. Inside of there is where implementation details live, rather than within consumers, which serves to centralize / hide complexity. If things need to change, we change them there rather than at each call-site.